app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    user_input = command.split()
+    if len(user_input) == 3:
+        filename, directories = user_input[1], user_input[2]
+
+        if "/" in directories:
```

Comment:
what if it is another OS? It's recommended to use `os.path.sep` for cross-platform compatibility.